11-payment-encoding.md

```diff
@@ -162,6 +166,11 @@ A reader MUST skip over unknown fields, an `f` field with unknown
 A reader MUST check that the SHA-2 256 in the `h` field exactly
 matches the hashed description.
 
+A reader MUST use a greater value for the `cltv_expiry` of the last
```

Comment:
Feels like this section should instead be placed somewhere in #253 as this section currently defines what implementers should note when _parsing_.

Reply:
Agreed, I removed it

Reply:
I also added a section in the rationale and took inspiration in [this comment of yours](https://github.com/lightningnetwork/lightning-rfc/pull/252#issuecomment-335260301)